app/src/main/java/org/worldbank/transport/driver/staticmodels/DriverApp.java

```diff
@@ -318,7 +317,7 @@ public static SecureDexClassLoader getSchemaClassLoader() {
      * Helper to revert to the backup model classes, if updates not found or could not be loaded.
      */
     public void loadBackupSchema() {
-        if (loadSchemaClasses(BACKUP_JAR_NAME, BACKUP_JAR_SCHEMA_VERSION)) {
+        if (loadSchemaClasses(BACKUP_JAR_NAME, getString(R.string.backup_jar_schema_version))) {
```

Comment:
This string was in code and not in the `strings` file because at runtime it would fail to resolve it before attempting to load the file. It would be worth testing if this works.

Reply:
So in both functions where `loadSchemaClasses` is called, there are other references to `R.string` that precede the calls to `loadSchemaClasses`, so it doesn't _seem_ like this would be an issue.

In any case, I tested out the two code paths that call `loadSchemaClasses` and was able to reference `R.string.backup_jar_schema_version` without a runtime error for both.